.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,10 +3,9 @@ If your package needs to be released within one or more release streams, and/or
 
 Supported Versions:
 
- * Nightly
- * 2.0
- * 1.24
- * 1.23
+ * develop
+ * 3.8
+ * 3.7
```

Comment:
We also still support 3.6.
```suggestion
 * 3.7
 * 3.6
```

Reply:
Even after the release of 3.8? As of right now, do you support 3.8 in RC-phase?

Reply:
> Even after the release of 3.8?

No

> As of right now, do you support 3.8 in RC-phase?

For end users: no, but it is a valid PR target.

Then again, given how old this list is and nobody had an issue with it, perhaps we should drop it and instead link to some place where there is a list of supported versions.